docs/pro/pricing.md

```diff
@@ -4,3 +4,271 @@ title: Alaveteli Pro Pricing
+`config/general.yml`. When you edit this file, remember it must be in the <a
+href="http://yaml.org">YAML syntax</a>. It's not complicated but &mdash;
+especially if you're editing a list &mdash; be careful to get the indentation
+correct. If in doubt, look at the examples already in the file, and don't use
+tabs.
+
+<code><a href="#enable_pro_pricing">ENABLE_PRO_PRICING</a></code>
+<br> <code><a href="#stripe_publishable_key">STRIPE_PUBLISHABLE_KEY</a></code>
+<br> <code><a href="#stripe_secret_key">STRIPE_SECRET_KEY</a></code>
+<br> <code><a href="#stripe_namespace">STRIPE_NAMESPACE</a></code>
+<br> <code><a href="#stripe_webhook_secret">STRIPE_WEBHOOK_SECRET</a></code>
+<br> <code><a href="#pro_referral_coupon">PRO_REFERRAL_COUPON</a></code>
+
+---
+
+## All the Pro Pricing settings
+
+<dl class="glossary">
+  <dt>
+    <a name="enable_pro_pricing"><code>ENABLE_PRO_PRICING</code></a>
+  </dt>
+  <dd>
+    Setting this will enable the Pro Pricing feature for all users.
+
+    <div class="more-info">
+      <p>Example:</p>
+      <ul class="examples">
+        <li><code>ENABLE_PRO_PRICING: true</code></li>
+      </ul>
+    </div>
+  </dd>
+
+  <dt>
+    <a name="stripe_publishable_key"><code>STRIPE_PUBLISHABLE_KEY</code></a> &amp;
+    <a name="stripe_secret_key"><code>STRIPE_SECRET_KEY</code></a>
+  </dt>
+  <dd>
+    API keys for your Stripe account. These can be found in the Stripe.com
+    developer dashboard.
+
+    <div class="more-info">
+      <p>Examples:</p>
+      <ul class="examples">
+        <li><code>STRIPE_PUBLISHABLE_KEY: pk_test_6rCXFw20WPjhMArF2Y3JquI2</code></li>
+        <li><code>STRIPE_SECRET_KEY: sk_test_pLEJl474WdGWt5Pz8KCUZo3F</code></li>
+      </ul>
+    </div>
+  </dd>
+
+  <dt>
+    <a name="stripe_namespace"><code>STRIPE_NAMESPACE</code></a>
+  </dt>
+  <dd>
+    An optional Stripe.com namespace which allows plans & coupons to be
+    separated from other resources within Stripe. If used the Stripe resources
+    will need IDs like: '&lt;namespace&gt;-&lt;id&gt;'
+
+    <div class="more-info">
+      <p>Example:</p>
+      <ul class="examples">
+        <li><code>STRIPE_NAMESPACE: alaveteli</code></li>
+      </ul>
+    </div>
+  </dd>
+
+  <dt>
+    <a name="stripe_webhook_secret"><code>STRIPE_WEBHOOK_SECRET</code></a>
+  </dt>
+  <dd>
+    Webhook key for your Stripe account. These can be found in the Stripe.com
+    developer dashboard.
+
+    <div class="more-info">
+      <p>Example:</p>
+      <ul class="examples">
+        <li><code>WEBHOOK_SECRET: wh_test_UD6BDsARFZIYb8273dbdl</code></li>
+      </ul>
+    </div>
+  </dd>
+
+  <dt>
+    <a name="pro_referral_coupon"><code>PRO_REFERRAL_COUPON</code></a>
+  </dt>
+  <dd>
+    <p>A Stripe coupon code – displayed to existing Pro users on their
+    subscriptions page – that they can share with friends for their friends to
+    receive a signup discount.</p>
+
+    <p>This should <strong>not</strong> include the `STRIPE_NAMESPACE`.</p>
```

Comment:
I think this (and  \`humanized_terms\`) should be `<code>` instead of Markdown backticks?